topiary-cli/src/error.rs

```diff
@@ -18,6 +18,7 @@ pub enum TopiaryError {
 pub enum CLIError {
     IOError(io::Error),
     Generic(Box<dyn error::Error>),
+    Multiple,
```

Comment:
Would it be possible to have `Multiple(Vec<CLIError>)`, or some other collection?

Reply:
Maybe... It would need to be `Multiple(Vec<TopiaryError>)` because the errors could be from the library. That shouldn't be a problem, but then what would we do with it?

I'll add this as a feature issue, along with some other thoughts I have around multiple error handling :+1:

Reply:
Added as #602